docs/source/development/widget.md

```diff
@@ -251,10 +250,215 @@ See [Storybook](https://6.docs.plone.org/storybook) with available widgets.
+    ```
+
 ```{note}
-Please contribute to this section!
+We use a `semantic-ui-react` component for our example.
+This is optional.
+`semantic-ui-react` is already installed in a standard Volto project.
+If it's not in your project, you may want to use alternatives or just don't have the need of a UI components library.
+```
+
+After a restart of Volto, your widget is now applied to all boolean widgets.
+You can see an example form with a boolean field in the following example.
+
+```{code-block} jsx
+:emphasize-lines: 27
+<Form
+    schema={{
+        fieldsets: [
+            {
+                id: 'default',
+                title: 'Default',
+                fields: [
+                    'fullname',
+                    'from',
+                    'subscribe_to_newsletter',
+                ],
+            },
+        ],
+        properties: {
+            fullname: {
+                title: intl.formatMessage(messages.fullnameTitle),
+                placeholder: intl.formatMessage(messages.fullnameTitle),
+            },
+            from: {
+                title: intl.formatMessage(messages.emailTitle),
+                placeholder: intl.formatMessage(messages.emailTitle),
+                type: 'email',
+                widget: 'email',
+            },
+            subscribe_to_newsletter: {
+                title: 'Subscribe to newsletter',
+                type: 'boolean',
+                default: false,
+            },
+        },
+        required: ['fullname', 'from',],
+    }}
+    onSubmit={onSubmit}
+    resetOnCancel={true}
+/>
+```
+
+
+
+### If you want to register the widget for arbitrary fields instead or additionally:
+
+```{code-block} jsx
+:emphasize-lines: 5
+import { ToggleWidget } from './ToggleWidget';
+
+const applyConfig = (config) => {
+    // …
+    config.widgets.widget.toggle_widget = ToggleWidget;
+    // …
+    return config;
+};
+export default applyConfig;
+```
+
+This widget named 'toggle_widget' can now be used as follows:
+
+
+```{code-block} jsx
+:emphasize-lines: 27-28
+<Form
+    schema={{
+        fieldsets: [
+            {
+                id: 'default',
+                title: 'Default',
+                fields: [
+                    'fullname',
+                    'from',
+                    'subscribe_to_newsletter',
+                ],
+            },
+        ],
+        properties: {
+            fullname: {
+                title: intl.formatMessage(messages.fullnameTitle),
+                placeholder: intl.formatMessage(messages.fullnameTitle),
+            },
+            from: {
+                title: intl.formatMessage(messages.emailTitle),
+                placeholder: intl.formatMessage(messages.emailTitle),
+                type: 'email',
+                widget: 'email',
+            },
+            subscribe_to_newsletter: {
+                title: 'Subscribe to newsletter',
+                type: 'pipapo',
+                widget: toggle_widget
```

Comment:
I believe that at the end of this line there should be a comma `,`.

Reply:
thanks for reviewing! fixed.